test/suites/test_dml.py

```diff
@@ -16,7 +17,13 @@ def setUpClass(self):
         self.srv = TarantoolServer()
         self.srv.script = 'test/suites/box.lua'
         self.srv.start()
-        self.con = tarantool.Connection(self.srv.host, self.srv.args['primary'])
+
+        args = [self.srv.host, self.srv.args['primary']]
+        self.con = tarantool.Connection(*args)
+        self.con_encoding_utf8 = tarantool.Connection(*args, encoding='utf-8')
+        self.con_encoding_none = tarantool.Connection(*args, encoding=None)
+        self.conns = [self.con, self.con_encoding_utf8, self.con_encoding_none]
```

Comment:
The connections are not closed in `tearDownClass()`.

Reply:
Fixed, thanks